packages/theming/apple-theme/src/appleSemanticColors.macos.ts

```diff
@@ -0,0 +1,178 @@
+import { ColorValue } from '@fluentui-react-native/theme-types';
+import { PlatformColor } from 'react-native';
+
+/** Palette of Apple Platform Colors, defined at https://developer.apple.com/documentation/appkit/nscolor/ui_element_colors */
+export interface AppleSemanticPalette {
+  /* Label Colors */
+
+  /** The primary color to use for text labels */
+  labelColor: ColorValue;
+
+  /** The secondary color to use for text labels */
+  secondaryLabelColor: ColorValue;
+
+  /** The tertiary color to use for text labels */
+  tertiaryLabelColor: ColorValue;
+
+  /** The quaternary color to use for text labels and separators */
+  quaternaryLabelColor: ColorValue;
+
+  /* Text Colors */
+
+  /** The color to use for text */
+  textColor: ColorValue;
+
+  /** The color to use for placeholder text in controls or text views */
+  placeholderTextColor: ColorValue;
+
+  /** The color to use for selected text */
+  selectedTextColor: ColorValue;
+
+  /** The color to use for the background area behind text */
+  textBackgroundColor: ColorValue;
+
+  /** The color to use for the background of selected text */
+  selectedTextBackgroundColor: ColorValue;
+
+  /** The color to use for the keyboard focus ring around controls */
+  keyboardFocusIndicatorColor: ColorValue;
+
+  /** The color to use for selected text in an unemphasized context */
+  unemphasizedSelectedTextColor: ColorValue;
+
+  /** The color to use for the text background in an unemphasized context */
+  unemphasizedSelectedTextBackgroundColor: ColorValue;
+
+  /* Content Colors */
+
+  /** The color to use for links */
+  linkColor: ColorValue;
+
+  /** The color to use for separators between different sections of content */
+  separatorColor: ColorValue;
+
+  /** The color to use for the background of selected and emphasized content */
+  selectedContentBackgroundColor: ColorValue;
+
+  /** The color to use for selected and unemphasized content */
+  unemphasizedSelectedContentBackgroundColor: ColorValue;
+
+  /* Menu Colors */
+
+  /** The color to use for the text in menu items */
+  selectedMenuItemTextColor: ColorValue;
+
+  /* Table Colors */
+
+  /** The color to use for the optional gridlines, such as those in a table view */
+  gridColor: ColorValue;
+
+  /** The color to use for text in header cells in table views and outline views */
+  headerTextColor: ColorValue;
+
+  /** The colors to use for alternating (odd Rows) content, typically found in table views and collection views */
+  alternatingOddContentBackgroundColor: ColorValue;
+
+  /** The colors to use for alternating (even Rows)content, typically found in table views and collection views */
+  alternatingEvenContentBackgroundColor: ColorValue;
```

Comment:
Like we discussed offline, I couldn't find these colors in the Apple doc. Should we remove them or just map them to alternatingContentBackgroundColor?

Reply:
The choice to separate the one macOS semantic color into two react native PlatformColors was done by whoever designed the PlatformColor API for react-native-macos. The intention of the `appleSemanticColors.<platform>.ts` files were to simply expose all the available PlatformColors, regardless of if we mapped it to any Token or Palette or not. The mappings of AppleColor->Palette happen in `appleTheme.colors.<platform>ts`.

Rather than remove the two colors from here, I would rather we choose not to use them in `appleTheme.colors.macos.ts` and use something else for list backgrounds, since it's confusing.